src/domains/issue/dto/index.ts

```diff
@@ -0,0 +1,103 @@
+export class IssueDTO {
+  id: string;
+  providerIssueId: string;
+  authorId: string | null;
+  assigneeId: string | null;
+  projectId: string;
+  stageId: string | null;
+  issueLabelId: string | null;
+  name: string;
+  title: string;
+  description: string | null;
+  priority: Priority;
+  storyPoints: number | null;
```

Comment:
story points should be a string, because it can generate an Exception (E.g. T-shirt size)